lib/temporal/workflow/context.rb

```diff
@@ -362,6 +369,11 @@ def schedule_command(command)
         state_manager.schedule(command)
       end
 
+      # @param query [Temporal::Api::Query::V1::WorkflowQuery]
+      def process_query(query)
+        dispatcher.dispatch(History::EventTarget.workflow, "queried:#{query.query_type}", query.query_args)
```

Comment:
This will only ever match the WILDCARD query handler. Why are you building this string with a "queried:" prefix to it?